ssh.go

```diff
@@ -249,6 +220,10 @@ func (s *SSH) setup() error {
 			return fmt.Errorf("public key lookup func is not provided")
 		}
 
+		if s.ReposForKeyLookupFunc == nil {
+			log.Println("no repository callback, an authorized user may access any repositories")
```

Comment:
Message to let the user know that once authenticated, anything goes. I think this is inline with how it currently works. Unless you overwrite a bunch of things and supply your own ssh config.